pkg/storage/cacher_test.go

```diff
@@ -287,7 +286,6 @@ func TestFiltering(t *testing.T) {
 	if err != nil {
 		t.Fatalf("Incorrect resourceVersion: %s", fooCreated.ResourceVersion)
 	}
```

Comment:
Since you are no longer changing initialVersion, please remove parsing it and use fooCreated.ResourceVersion instead